events_ingress/src/test/java/com/adobe/aio/event/publish/feign/FeignPublishServiceTestDrive.java

```diff
@@ -28,7 +28,7 @@ public class FeignPublishServiceTestDrive {
 
   public static void main(String[] args) {
     try {
-      Workspace workspace = WorkspaceUtil.getSystemWorkspaceBuilder().build();
+      Workspace workspace = WorkspaceUtil.getWorkspaceBuilder().build();
       String providerId = WorkspaceUtil.getSystemProperty(AIO_PROVIDER_ID);
```

Comment:
don't you want to load these other ` AIO_*` property using the same pattern ?  and augment `getSystemProperty` with `System.getEnv()` ?

Reply:
ah sure!

Reply:
done